posteriors/utils.py

```diff
@@ -151,6 +152,31 @@ def fvp(
     More info on empirical Fisher matrices can be found in
     [Martens, 2020](https://jmlr.org/papers/volume21/17-678/17-678.pdf).
 
+    Examples:
+        ```python
+        from functools import partial
```

Comment:
not sure the imports are useful in such examples?

Reply:
I think its good, its important to know where `partial` in particular comes from for those who aren't familiar